modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.scss

```diff
@@ -20,3 +20,34 @@
 		z-index: 1;
 	}
 }
+
+.lfr-objects__card {
+	margin: 1.5rem 0 0 !important;
+}
+
+@media (max-width: 1440px) {
+	.lfr__notification-template {
+		&-cards {
+			margin-left: 10rem;
+			margin-right: 10rem;
+		}
+	}
+}
+
+@media (max-width: 1200px) {
+	.lfr__notification-template {
+		&-cards {
+			margin-left: 5rem;
+			margin-right: 5rem;
+		}
+	}
+}
+
+@media (max-width: 620px) {
+	.lfr__notification-template {
+		&-cards {
+			margin-left: 1.5rem;
+			margin-right: 1.5rem;
+		}
+	}
+}
```

Comment:
Hey @alberttgomes, to avoid this code repetition we can create a function in `SASS` to do this, something like:

```scss
@mixin responsiveCards($maxWidth, $margin) {
	@media (max-width: #{$maxWidth}) {
		.lfr__notification-template {
			&-cards {
				margin-left: #{$margin};
				margin-right: #{$margin};
			}
		}
	}
}
```

And then we call this function passing the resolution and the margin value using the `@include` annotation, I think we can use for the most used resolutions. like: 

```scss
@include responsiveCards(3840px, 80rem);
@include responsiveCards(2160px, 29rem);
@include responsiveCards(1920px, 21rem);
@include responsiveCards(1600px, 11rem);
@include responsiveCards(1280px, 5rem);
```
And with that you can remove the code from lines `6-9` which will be covered by this code.

Reply:
@danvb112 ok, dani!!
I will make the changes!!